test/framework/easyblock.py

```diff
@@ -147,6 +146,64 @@ def extra_options():
         eb.close_log()
         os.remove(eb.logfile)
 
+    def test_load_module(self):
+        """Test load_module method."""
+        # copy OpenMPI module used in gompi/1.3.12 to fiddle with it, i.e. to fake bump OpenMPI version used in it
+        tmp_modules = os.path.join(self.test_prefix, 'modules')
+        mkdir(tmp_modules)
+
+        test_dir = os.path.abspath(os.path.dirname(__file__))
+        copy_dir(os.path.join(test_dir, 'modules', 'OpenMPI'), os.path.join(tmp_modules, 'OpenMPI'))
+
+        openmpi_module = os.path.join(tmp_modules, 'OpenMPI', '1.6.4-GCC-4.6.4')
+        ompi_mod_txt = read_file(openmpi_module)
+        write_file(openmpi_module, ompi_mod_txt.replace('1.6.4', '2.0.2'))
+
+        self.modtool.use(tmp_modules)
+
+        orig_tmpdir = os.environ.get('TMPDIR')
```

Comment:
Hmm, good point...

It seems that the default Travis environment indeed causes it to be long enough, otherwise the `assertTrue` on the warning would fail, but we better make sure the `$TMPDIR` value is long enough, I'll fix that, thanks.